src/style.css

```diff
@@ -0,0 +1,63 @@
+@font-face {
+  font-family: "avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+body {
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
if you want apply `box-sizing: border-box;` to every element, you should set it to the `* `selector